modules/opensearch/opensearch.info

```diff
@@ -0,0 +1,6 @@
+name = Opensearch
+description = A OpenSearch search-provider.
```

Comment:
For clarity mentioning DBC, https://opensource.dbc.dk/services/open-search-web-service or something like that would be useful.